rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -109,6 +109,8 @@ def get_all_info(
         - RemoteError
         """
         result, total_nfts_num = self._get_all_nft_data(addresses, ignore_cache=ignore_cache)
+        self._filter_ignored_nfts(result)
+
```

Comment:
Why not adding this logic inside `_get_all_nft_data`?

Reply:
The only reason I didn't add it there was because `get_all_nft_data()` is cached

Reply:
ah okey. Good point